dashboard/test/controllers/aichat_controller_test.rb

```diff
@@ -42,44 +50,44 @@ class AichatControllerTest < ActionController::TestCase
 
   test 'pilot teacher has access test' do
     sign_in(@genai_pilot_teacher)
-    post :chat_completion, params: @valid_params
+    post :chat_completion, params: @valid_params, as: :json
```

Comment:
This took me a bit to debug after changing the default `storedMessages` param to an empty array (an empty array is what gets sent by the front end for the first message when a user opens the page, so seems like a sensible default).

Related to an earlier change Alice had to make to handle an empty array passed as a param in the aichat controller, I think this comment describes the issue:
https://github.com/rails/rails/issues/27215#issuecomment-263972010

Reply:
Oh nice! Thanks for sharing the comment.